src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,197 @@
+package duke;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+/**
+ Class to process the Tasklist of the chatbot
+ */
+public class TaskList {
+
+    /**
+     * Prints the arraylist for the user of the chatbot
+     *
+     * @param storeList arraylist to be printed
+     * @return String of the contents of the list
+     *
+     */
+    public static String printTheList(ArrayList<Task> storeList) {
+        StringBuffer sb = new StringBuffer();
+        for (Task s : storeList) {
+            sb.append(s.toString());
+            sb.append(" ");
+        }
+
+        String str = sb.toString();
+        return str;
+    }
+
+    /**
+     * Deletes a specific item in the tasklist
+     *
+     * @param command command of which task to delete
+     *
+     * @return String of response
+     *
+     */
+    public static String deleteTask(String command, ArrayList<Task> storeList) {
+        if (command.length() <= 7) {
+            return "Master, you wished wrongly. Remember you have to wish in this format " +
+                    "delete tasknumber. Please wish again";
+        } else {
+            try {
+                Task t = storeList.get(Integer.parseInt(command.substring(7)));
```

Comment:
maybe it would be clearer to rename the variable "t" to something more meaningful